.gitignore

```diff
@@ -0,0 +1,2 @@
+slice.go
```

Comment:
remove files if not necessary, we add configuration files in git ignore all other unwanted files can be removed